wdl/import.json

```diff
@@ -0,0 +1,16 @@
+{
+    "pheweb_import.summaryfiles":"gs://r2_data/pheweb_files.txt",
+    "pheweb_import.docker": "gcr.io/finngen-refinery-dev/pheweb-import:4",
+    "pheweb_import.disktype": "SSD",
```

Comment:
We should stop giving resources as WDL inputs because call caching will give a miss if the only difference is e.g. disk type or # CPUs